lib/Controller/FolderController.php

```diff
@@ -73,7 +78,12 @@ private function formatFolder(array $folder): array {
 	#[NoAdminRequired]
 	#[ApiRoute(verb: 'GET', url: '/folders')]
 	public function getFolders(bool $applicable = false): DataResponse {
-		$folders = $this->manager->getAllFoldersWithSize($this->getRootFolderStorageId());
+		$storageId = $this->getRootFolderStorageId();
+		if ($storageId === null) {
+			throw new OCSNotFoundException();
```

Comment:
Should be documented no? Isn’t the openapi extractor checking that?

Reply:
Yeah, but I will use a psalm rule for that to enforce documenting all possible exceptions. So expect a follow up for that.